### PR TITLE
Improve Chinese Translation

### PR DESCRIPTION
**Pull request by xiaopangju**
According to https://docs.github.com/cn/github/collaborating-with-issues-and-pull-requests , change "提案" into "拉取请求".

### CODE REVIEW

**Comment by xiaopangju** [comment]
@anuraghazra Why are there two mistakes here?
图片

**Comment by xiaopangju** [comment]
@liyishuai Can you help me review this? Thanks!

File: src/translations.js [hunk 1]
```diff
@@ -75,7 +75,7 @@ const statCardLocales = ({ name, apostrophe }) => {
       pl: "Wszystkie commity",
     },
     "statcard.prs": {
-      cn: "提案数（PR）",
+      cn: "拉取请求数（PR）",
```
Comment (liyishuai) [inline src/translations.js:78]: 如不知英文pull request，见到”拉取请求数“一词会理解为“number of requests pulled”，故主张激进地意译为“提案”。特别是在有英文标注的情况下，没有直译的必要。

**Review by liyishuai — CHANGES_REQUESTED** [review]

File: src/translations.js (hunk 1 above)
Comment (xiaopangju) [reply src/translations.js:78]: 那么您认为关于这个可以保留原文吗？但这样的话，就与上面和下面的翻译风格（对于特定名词中文译文加上英文原文）不一样了。感觉很难取舍

Comment (liyishuai) [reply src/translations.js:78]: 目前统一的体例是意译+原文。鉴于你提出的改动与官方文档一致，可先采纳，待官方文档改进后再另行修订。

**Review by liyishuai — APPROVED** [review]
@anuraghazra While I have linguistic concerns, the changes here are coherent with GitHub documentations. Worth merging now and revising after GitHub documents changed.

File: src/translations.js (hunk 1 above)
Comment (antoniosuperba) [reply src/translations.js:78]: https://p2guides.wordpress.com/

**Comment by xiaopangju** [comment]
please keep open

**Comment by anuraghazra** [comment]
Thanks for the changes @xiaopangju and @liyishuai for review

**Pull request merged**